ui/admin/app/components/form/target/details/index.hbs

```diff
@@ -224,21 +224,10 @@
       {{/if}}
       <F.Control>
         {{#if this.showTargetFilters}}
-          {{#if this.workerDiagramHCPDualFilter}}
-            <WorkerDiagram::DualFilter::Hcp
-              @egressFilter={{this.egressWorkerFilterEnabled}}
-              @ingressFilter={{this.ingressWorkerFilterEnabled}}
-            />
-          {{else if this.workerDiagramDualFilter}}
-            <WorkerDiagram::DualFilter
-              @egressFilter={{this.egressWorkerFilterEnabled}}
-              @ingressFilter={{this.ingressWorkerFilterEnabled}}
-            />
-          {{else}}
-            <WorkerDiagram::SingleFilter
-              @egressFilter={{this.egressWorkerFilterEnabled}}
-            />
-          {{/if}}
+          <WorkerDiagram
+            @egressFilter={{this.egressWorkerFilterEnabled}}
+            @ingressFilter={{this.ingressWorkerFilterEnabled}}
```

Comment:
It would be nice if the argument names matched and indicated that these are booleans, e.g. `this.egressWorkerFilterEnabled`.